### PR TITLE
User timeline

### PR DESCRIPTION
**Pull request by AWinterman**
Adds an http resource and sql queries for a user-level timeline of events.

### CODE REVIEW

File: connect-examples/node-connect-postgres/README.md [hunk 1]
```diff
@@ -1,13 +1,24 @@
...
+An example of getting connect data into postgres, and of one possible analytics
+application built on top of the tables that result. It's probably not suitable
+for a big production app because it inserts individual events into postgres as
+it receives them.
 
 I tested it with an AWS RDS PostgreSQL instance of the `db.t2.micro` class. 
 
+If you're reading the code, I recommend starting with [./schema.sql](./schema.sql), which
+defines the tables we expect to be in PostgreSQL, and then moving on to 
+[./index.js](./index.js), which is the entry point for the project. `npm start` essentiall
```
Comment (fardog) [inline connect-examples/node-connect-postgres/README.md]: typo: `essentiall`

File: connect-examples/node-connect-postgres/README.md [hunk 2]
```diff
@@ -1,13 +1,24 @@
...
+runs `node index.js`.
+
 ## Configuration, installation and running
 
-It's a node project, so from this directory, run `npm i` to install its
+It's a node project, so install node either via whatever package manager you like,
```
Comment (fardog) [inline connect-examples/node-connect-postgres/README.md]: nit: `via your preferred package manager` reads better here

File: connect-examples/node-connect-postgres/README.md [hunk 3]
```diff
@@ -31,3 +42,85 @@ npm start
 
 I've included a convenience npm script for connecting to the postgres instance
 configured with your environment variables; namely `npm run connect`
+
+## HTTP Service
+
+Running `npm start` also starts an HTTP service which provides a JSON REST API
+for a pair of particular queries against this postgres database. 
+
+> NB: It's clearly sub-optimal to have the read and write services both running
```
Comment (fardog) [inline connect-examples/node-connect-postgres/README.md]: nit: remove `clearly`

File: connect-examples/node-connect-postgres/lib/server.js [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+        }
+
+        stream 
+          .on('error', failure)
+
+        stream.pipe(display).pipe(stringify()).pipe(res)
+
+        stream.end(u.query)
+
+        function success(data) {
```
Comment (fardog) [inline connect-examples/node-connect-postgres/lib/server.js]: this is never used

**Comment by AWinterman** [comment]
👋

**Pull request closed without merging**